content/connectors/sinks/dynamodb.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Dynamodb
+section: Sink
+---
+
+## Overview
+
+The Dynamodb Sink Connector is a sink connector which reads events from a
+fluvio topic, deserializes them as json and inserts those key value pairs based
+on the columns in the config.
+
+## Configuration Options
+
+%copy%
+```yaml
+# dynamodb-connector.yml
+name: dynamodb-connector
+type: dynamodb-sink
+topic: dynamodb-test
+create_topic: true
+parameters:
+  table-name: dynamodb-test
+  column-names: col_1,col_2,col_3
+  column-types: N,N,S
+secrets:
+  AWS_REGION: "us-west-2"
+  AWS_ACCESS_KEY_ID: ""
+  AWS_SECRET_ACCESS_KEY: ""
+```
+
+### Parameters
+
+* `table-name` - The name of the [dynamodb table name]
+* `column-names` - A comma separated list of the keys. **The first key is the partition key**
+* `column-types` - A comma separated list of the types for the keys.
```

Comment:
Is there a reference guide on `column types` allowed?

Reply:
I added it.